tests/common/mellanox_data.py

```diff
@@ -122,6 +185,15 @@
         "psus": {
             "number": 2,
             "hot_swappable": True
+        },
+        "cpu_pack": {
+            "number": 1
+        },
+        "cpu_cores": {
+            "number": 2
+        },
+        "ports":{
+            "number": 32
         }
     },
     "ACS-MSN3510": {
```

Comment:
IMO this system is not part of the supported devices. Please recheck and remove unsupported systems,.

Reply:
3510 not supported yet, removed.